src/services/API/sqs.js

```diff
@@ -5,12 +5,12 @@ const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
 const sqsClient = new SQSClient({
   region: REGION,
   credentials: {
-    accessKeyId: 'AKIARUBOPCTSSU3W7SML',
-    secretAccessKey: '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',
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_KEY,
   },
 });
 
-async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist) {
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType) {
     const messageBody = [
       {
         DateTime: Date.now().toString(),
```

Comment:
I think it'll be best to convert these into camel case?
As I've commented on the lambda PR as well, would be best to use `camelCase` here. Also because you'll be accessing these values from the SQS payload na, so would have to change there as well.

https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase#:~:text=The%20JSON%20syntax%20does%20not,or%20snake_case%20should%20work%20fine. 
Read the second answer, pretty interesting!

Reply:
Also, did you add it to github secrets?

Reply:
Also some merge commits are there, can you look at them too?